### PR TITLE
OT Reconstruction: fix warning unused variable

### PR DESCRIPTION
**Pull request by sgiraudot**
## Summary of Changes

Fix unused variable.

## Release Management

* Affected package(s): Optimal_transportation_reconstruction_2
* Issue(s) solved (if any): fix https://github.com/CGAL/cgal/issues/2111

### CODE REVIEW

File: Optimal_transportation_reconstruction_2/include/CGAL/Optimal_transportation_reconstruction_2.h [hunk 1]
```diff
@@ -989,7 +989,6 @@ class Optimal_transportation_reconstruction_2
   {
     int i = f->index(v);
     Face_handle cf = fmap[f];
-    Vertex_handle cv = vmap[v];
```
Comment (afabri) [inline Optimal_transportation_reconstruction_2/include/CGAL/Optimal_transportation_reconstruction_2.h]: Note that now vmap is an unused variable.

**Comment by sgiraudot** [comment]
(Okay, for the last remark I just silenced the parameter, to avoid breaking the API in case this function is used elsewhere.)

**Comment by afabri** [comment]
And did you check that it is not an error to not use the parameter?
Is the function documented? If not I would clean the API.

**Comment by sgiraudot** [comment]
As far as I understand, it's not an error. And it's not documented. I'll fix it.

**Comment by lrineau** [comment]
The issue is in 4.10. Would it be difficult to backport the PR to 4.10-branch?

**Comment by sgiraudot** [comment]
It shouldn't, it's very self contained and the code hasn't changed since 4.10. What should I do?

**Comment by lrineau** [comment]
The commands are:
```shell
# preparation:
git fetch cgal
git checkout Optimal_transportation_reconstruction_2-Fix_warning_unused_variable-GF

# then the actual rebase:
git rebase cgal/master --onto cgal/releases/CGAL-4.10-branch

# and the push, with `--force` because the history was rewrote
git push --force mine Optimal_transportation_reconstruction_2-Fix_warning_unused_variable-GF
```

**Comment by sgiraudot** [comment]
Okay, it's done.

**Comment by lrineau** [comment]
Great, thanks.

**Comment by sgiraudot** [comment]
(It looks like Travis failed for an `apt-get` error not related to this PR.)

**Comment by lrineau** [comment]
I have restarted the failed sub-job. It will eventually become green.

**Pull request merged**